tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -1137,5 +1138,107 @@ public void CheckResouceIsOverridden ([Values (true, false)] bool useAapt2)
 			}
 		}
 
+		[Test]
+		[Category ("WearOS")]
+		public void DotNetInstallAndRunPreviousSdk ([Values (false, true)] bool isRelease)
+		{
+			XamarinAndroidApplicationProject proj;
+				proj = new XamarinFormsAndroidApplicationProject {
+					TargetFramework = "net7.0-android",
+					IsRelease = isRelease,
+					EnableDefaultItems = true,
+				};
```

Comment:
Did something copy/paste weird? Seems indented extra, and maybe we can set `proj` when declaring it.

Reply:
Thanks yeah it was a combination of a bad cp + edit.